<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UDPClass</class>
 <widget class="QMainWindow" name="UDPClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>404</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>590</width>
    <height>404</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>590</width>
    <height>404</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>UDP</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextBrowser" name="dialogBrowser">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>441</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="dialogEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>300</y>
      <width>441</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="sendButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>340</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>发送消息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="flushButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>300</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>刷新列表</string>
    </property>
   </widget>
   <widget class="QLabel" name="MyIpLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QListWidget" name="IpListWidget">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>50</y>
      <width>151</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="chatIpLabel">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>0</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>联系人列表</string>
    </property>
   </widget>
   <widget class="QPushButton" name="appointIpButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>300</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>指定IP</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="appointIpEdit">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>30</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>506</x>
      <y>300</y>
      <width>20</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="9"/>
 <resources>
  <include location="udp.qrc"/>
 </resources>
 <connections/>
</ui>
